KenticoOnboardingApplication/test/KenticoOnboardingApplication.Api.Tests/Controllers/ListControllerIntegrationTest.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using KenticoOnboardingApplication.Api.Controllers;
+using KenticoOnboardingApplication.Contracts.Helpers;
+using KenticoOnboardingApplication.Contracts.Models;
+using KenticoOnboardingApplication.Contracts.Repositories;
+using KenticoOnboardingApplication.Tests.Base;
+using KenticoOnboardingApplication.Tests.Base.Factories;
+using NSubstitute;
+using NUnit.Framework;
+using Unity;
+
+namespace KenticoOnboardingApplication.Api.Tests.Controllers
+{
+    [TestFixture]
+    public class ListControllerIntegrationTest
+    {
+        [Test]
+        public async Task PutItemAsync_ShouldCallRepositoryOnce()
+        {
+            var container = CreateContainer();
+            var controller = ResolveController(container);
+            var (repository, timeManager) = ResolveRepositoryAndTimeManager(container);
+            var (originItem, expectedItem, itemToPut) = CreateOriginExpectedAndItemToPut();
+            ArrangeRepositoryAndTimeManager(repository, timeManager, originItem, expectedItem);
```

Comment:
I might rather go with `SetUp-` or `Configure-` since I believe `arrange` includes creation in this context and you have already created everything.